src/ResourceManager/PowerBI/Commands.PowerBI/Commands/GetAzureRmPowerBIEmbeddedCapacity.cs

```diff
@@ -0,0 +1,82 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.PowerBI.Models;
+using Microsoft.Azure.Commands.PowerBI.Utilities;
+using Microsoft.Azure.Management.PowerBIDedicated;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+
+namespace Microsoft.Azure.Commands.PowerBI
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmPowerBIEmbeddedCapacity"),
+        OutputType(typeof(List<PSDedicatedCapacity>))]
+    public class GetAzurePowerBIEmbeddedCapacity : PowerBICmdletBase
```

Comment:
@selasagi the naming of these cmdlets still confuses me - you are receiving a `DedicatedCapacity` object from the server and transforming it into a `PSDedicatedCapacity` object, but the cmdlet is called `Get-AzureRmPowerBIEmbeddedCapacity` instead of `Get-AzureRmPowerBIDedicatedCapacity`. I was under the impression that `PowerBIEmbedded` objects are separate from `PowerBIDedicated` objects.

Reply:
PSDedicatedCapacity changed to PSPowerBIEmbeddedCapacity